### ISSUE TITLE
ActiveDirectory module missing from Integrated environment in Powershell Universal 2.9.1

### ISSUE THREAD

**Issue by waikinw** [ironmansoftware/powershell-universal#1037]
### Steps to Reproduce

Unable to use ActiveDirectory module commands in Integrated environment in Powershell Universal 2.9.1.

### Expected behavior

```console
Need commands like Get-ADObject, Get-ADGroup at minimum for roles.ps1 scripts
```

### Actual behavior

```console
Missing module
```

### Environment data

PSU 2.9.1

**Comment by adamdriscoll** [comment]
The integrated environment is running PS 7.1 and should be able to load any module available to that environment. What version of the AD module do you have installed?

**Comment by waikinw** [comment]
Oddly, the module available in the PS 7.2.1 environment. Just not the integrated environment. It disappeared after going from 2.9.0 to 2.9.1.

It's ActiveDirectory version 1.0.1.0 - C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\ActiveDirectory\ActiveDirectory.psd1

Also, only have 20 modules in Integrated environment. 7.2.1 has 91 modules. 5.1 has 88-143 modules.

**Comment by adamdriscoll** [comment]
Strange. 

What happens if you import the module directly in your role script?

```
impo C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\ActiveDirectory\ActiveDirectory.psd1
```

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.ironmansoftware.com/t/powershell-universal-2-9-1/6747/11

**Comment by adamdriscoll** [comment]
This has been resolved in PSU 2.9.2
